Makefile

```diff
@@ -95,6 +95,7 @@ install: init ## install the package to the active Python's site-packages
 	pipenv run python setup.py install
 
 dev: init ## set up a development environment
+	pipenv --three
```

Comment:
Might be worth a comment here stating you only need this if python2 is set as the default for your system or you want to be extra sure that you are using python 3.